js_modules/dagster-ui/packages/ui-core/src/asset-graph/ComputeGraphData.worker.ts

```diff
@@ -0,0 +1,25 @@
+import {FeatureFlag} from 'shared/app/FeatureFlags.oss';
+
+import {computeGraphData} from './ComputeGraphData';
+import {ComputeGraphDataMessageType} from './ComputeGraphData.types';
+import {setFeatureFlags} from '../app/Flags';
+
+type WorkerMessageData = ComputeGraphDataMessageType;
+
+self.addEventListener('message', async (event: MessageEvent & {data: WorkerMessageData}) => {
+  const {data} = event;
+
+  if (data.type === 'computeGraphData') {
+    if (data.flagAssetSelectionSyntax) {
+      setFeatureFlags({[FeatureFlag.flagAssetSelectionSyntax]: true});
+    }
```

Comment:
What if they set it back to false? Do we need to sync the value back to false? (Not sure if this state is entirely reset on each run or if this worker runs idle in the background, I think it does?)

Reply:
We reload the page when feature flags change so no risk